src/qml/AboutPage.qml

```diff
@@ -29,6 +30,19 @@ Flickable {
         id: diskUsage
     }
 
+    Process {
+        id: process
```

Comment:
This is not particularly descriptive, if we plan to use Processes like this in this AboutPage there would probably be more so they would each have a more specific name